packages/web/src/components/Home/FeatureOptions.ts

```diff
@@ -1,37 +1,118 @@
-export interface Feature {
+interface Slide {
+    caption: string; // Caption above image
+    imgSrc: string; // Image src
+}
+
+interface Feature {
     id: number;
     text: string; // Left menu
-    caption: string; // Caption above image
-    imgSrc?: string; // Image src
+    slides: Slide[]; // Right carousel
 }
 
-export const features = [
+export default [
     {
         id: 1,
         text: "Find target files quickly",
-        caption: `Filter, sort, group or search for your files
-      in a predictable and standard way and view thumbnails for 
-      quick identification. Switch to thumbnail view for...this 
-      is placeholder text for now.`,
+        slides: [
+            {
+                imgSrc: "/FindingData/Group.png",
+                caption: `Unlike traditional file systems, BioFile Finder allows you to dynamically generate your folder structure based on the metadata of your files. In this screenshot, the files are organized by (Grouped by) the "Gene" and "Structure" columns found in the selected data source.`,
```

Comment:
I don't think we need to start with "Unlike traditional file systems," I would just start with BioFile Finder allows...